manifests/00-ingress-credentials-request.yaml

```diff
@@ -69,6 +71,7 @@ metadata:
     include.release.openshift.io/ibm-cloud-managed: "true"
     include.release.openshift.io/self-managed-high-availability: "true"
     include.release.openshift.io/single-node-developer: "true"
+    capability.openshift.io/name: Ingress
```

Comment:
Combine the annotation value with the existing annotation.

Reply:
Done.